apps/tutorial-servo-can-control/include/TutorialServo.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "mbed.h"
+
+class TutorialServo {
+
+public:
+    
+    // Constructor
+    // Takes in servo pin name, and optionally a servo range (default value of 180.0 degrees), 
+    // a min pulsewidth (default 1ms) and max pulsewidth (default 2ms)
+
+    TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2);
+
+    // Set servo position in degrees (ex. 45deg)
+    void setPositionInDegrees(const float degrees);
+
+    // Set servo position in percentage of range of motion (ex. 50%)
+    void setPositionInPercentage(const float percentage);
```

Comment:
i dont think this function was in the training? did u do 20/21

Reply:
Yeah since it's been a while so I have no idea why I added this function. I think it was just me playing around with the code a little but not sure how it ended in this pr.

Reply:
Also yes I did do the 20/21 training